src/nav/alerts-and-applied-intelligence.yml

```diff
@@ -29,8 +29,8 @@ pages:
             path: /docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-best-practices
       - title: Detect anomalies
         pages:
-          - title: Proactive detection
-            path: /docs/alerts-applied-intelligence/applied-intelligence/proactive-detection/proactive-detection-applied-intelligence
+          - title: Anomaly detection
```

Comment:
Okay now i get to the URL. :D I think the file itself will have to be renamed for this path to change (and in that case you'd also want to add a redirect)